src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml

```
@@ -0,0 +1,49 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/dataset.h5ad"
+info:
+  label: "Dataset"
+  summary: "The dataset to pass to a method."
+  slots:
+    layers: 
+      - type: integer
+        name: counts
+        description: Raw counts
+        required: true
+    obs:
+      - type: string
+        name: label
```

```suggestion
        name: cell_type
```
see https://vscode.dev/github/vishnu-vasan/openproblems-v2/blob/c2c_source_target/src/datasets/api/file_raw.yaml#L33